sdks/java/extensions/sql/jdbc/build.gradle

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply from: project(":").file("build_rules.gradle")
+applyJavaNature(failOnWarning: true, enableSpotless: true, shadowClosure: {})
+
+configurations {
+  // Run tests from shaded jar
+  shadowTestRuntimeClasspath
+  integrationTest
+}
+
+dependencies {
+  compile project(path: ":beam-sdks-java-extensions-sql", configuration: "shadow")
+  compile "jline:jline:2.14.6"
+  compile "sqlline:sqlline:1.3.0"
+  compile library.java.slf4j_jdk14
+  compileOnly library.java.findbugs_annotations
+  testCompileOnly library.java.findbugs_annotations
+  shadowTestRuntimeClasspath project(path: project.path, configuration: "shadowTest")
+  shadowTestRuntimeClasspath project(path: project.path, configuration: "provided")
+  integrationTest sourceSets.test.output
```

Comment:
Note that you should specifically list why your using the sourceSets output here as typically one depends on a project configuration like shadow to get the artifacts that it produces.